test/framework/run.py

```diff
@@ -543,7 +543,7 @@ def test_run_cmd_script(self):
         """Testing use of run_cmd with shell=False to call external scripts"""
         py_test_script = os.path.join(self.test_prefix, 'test.py')
         write_file(py_test_script, '\n'.join([
-            '#!/usr/bin/python',
+            '#!/usr/bin/env python',
```

Comment:
Is this sufficient? Should we also consider systems that only have `python3`, not `python`?

Reply:
As this is targeting `develop` we cannot assume that we are using `python3`

Reply:
True, so good as is